### ISSUE TITLE
code formater tool

### ISSUE THREAD

**Issue by maidamai0** [qCring/QuickVtk#51]
I have to disable auto-format function of my code editor when I modify code to keep the code style, a code formatter may help with this.

[Clang-tidy](https://clang.llvm.org/extra/clang-tidy/) is a good choice.

**Comment by qCring** [comment]
Thanks for your feedback! Do you mean the C++ code style of the project? Since I'm using Xcode on macOS, I can imagine that many other IDEs have issues with how Xcode handles .cpp/.hpp files...

**Comment by maidamai0** [comment]
Yes, I mean the c++ code style.According to [doc](https://clang.llvm.org/extra/clang-tidy/Integrations.html), it seems XCode does't have a integration with clang-tidy.

Clang-tidy can be [integrated with cmake](https://gitlab.kitware.com/cmake/cmake/issues/18926), you might want to have a try.
